bindings/python/iota_sdk/types/essence.py

```diff
@@ -38,24 +46,23 @@ class RegularTransactionEssence(TransactionEssence):
         network_id: The unique value denoting whether the block was meant for mainnet, shimmer, testnet, or a private network.
                     It consists of the first 8 bytes of the BLAKE2b-256 hash of the network name.
         creation_slot: The slot index in which the transaction was created.
-        context_inputs: The inputs that provide additional contextual information for the execution of a transaction.
```

Comment:
I don't know why this was changed as this is now wrong.

Reply:
But we already have https://github.com/iotaledger/iota-sdk/issues/1176 so I guess it's alright.